pkg/controller/account/account_controller.go

```diff
@@ -660,9 +660,41 @@ func (r *ReconcileAccount) initializeRegions(reqLogger logr.Logger, currentAcctI
 		return err
 	}
 
+	if !currentAcctInstance.IsBYOC() { // This is a normal account creation
+		reqLogger.Info("Normal account region initialization")
+		go r.asyncRegionInit(reqLogger, currentAcctInstance, creds, regionAMIs, castAWSRegionType(regionsEnabledInAccount.Regions))
+		return nil
+	}
+
+	accountClaim, acctClaimErr := r.getAccountClaim(currentAcctInstance)
+	if acctClaimErr != nil {
+		reqLogger.Info("Accountclaim not found")
+		return acctClaimErr
+	}
+
+	for _, wantedRegion := range accountClaim.Spec.Aws.Regions {
+		found := false
+		for _, enabledRegion := range regionsEnabledInAccount.Regions {
+			reqLogger.Info("Comparing regions", "wantedRegion", wantedRegion, "enabledRegion", enabledRegion)
```

Comment:
Do we want this log message for every region in the list on every CCS claim?

Reply:
No this was for debugging, thanks for the catch